README.md

```diff
@@ -40,27 +40,35 @@ Commands:
     Show commit logs.
 
   status
-    Show working-tree status. Also stage and commit changes.
+    Show working-tree status. Also, stage and commit changes.
 
 ```
 
 ## Configure
-- To set line size `export GITIN_LINESIZE=5`
+- To set the line size `export GITIN_LINESIZE=5`
 - To hide help `export GITIN_HIDEHELP=true`
 
 ## Development Requirements
-- Requires gitlib2 v27 and `git2go`. See the project homepages for build instructions.
-  1. download git2go; `go get -d gopkg.in/libgit2/git2go.v27`
-  2. make sure you have `cmake`, `pkg-confid` and `libssl-dev` installed
-  3. `cd` into `$GOPATH/src/gopkg.in/libgit2/git2go.v27`
-  4. initialize submodules by running `git submodule update --init`
-  5. change the libigt2 version to your version (in this case its 0.27) in the install script (`script/install-libgit2.sh`)
-  6. run the script `./script/install-libgit2.sh`
+- Requires gitlib2 v27 and `git2go`. See the project homepages for more information about build instructions. For gitin you can simply;
+  - macOS:
+    1. install libgit2 via `brew install libgit2` (consider that libgit2.v27 is required)
+  - Linux and macOS(if you want to build your own):
+    1. download git2go; `go get -d gopkg.in/libgit2/git2go.v27`
+    2. make sure you have `cmake`, `pkg-config` and `libssl-dev` installed
+    3. `cd` into `$GOPATH/src/gopkg.in/libgit2/git2go.v27`
+    4. initialize submodules by running `git submodule update --init`
+    5. change the libigt2 version to your version (in this case its 0.27) in the install script (`script/install-libgit2.sh`)
```

Comment:
I think this instruction is not clear enough, maybe: 
```suggestion
    5. change the libigt2 version to your version (in this case its 0.27) in the install script (e.g. `nano script/install-libgit2.sh` or `vim script/install-libgit2.sh`) and change `LG2VER` to 0.27.0
```

Reply:
thank you @ybbond  this is more clear